third_party/move/move-prover/boogie-backend/src/options.rs

```diff
@@ -221,7 +220,7 @@ impl BoogieOptions {
             add(&[&format!("-proverOpt:PROVER_PATH={}", &self.z3_exe)]);
         }
         if self.use_array_theory {
-            add(&["-useArrayTheory"]);
+            add(&["-useArrayAxioms"]);
```

Comment:
Is there a reference to introduce these flag changes in boogie?

Reply:
https://github.com/boogie-org/boogie/pull/732

Reply:
@junkil-park As discussed in https://github.com/boogie-org/boogie/discussions/769, `-useArrayAxioms` should be set by default apart from removing `-useArrayTheory`. And thus the option `use_array_theory` may also be removed. cc @wrwg

Reply:
Yes, the naming is confusing. 'useArrayTheory' == SMT arrays, 'useArrayAxioms' boogie arrays, which is our default.